dev/breeze/src/airflow_breeze/commands/release_candidate_command.py

```diff
@@ -295,44 +303,38 @@ def tarball_release(
 
 
 def create_tarball_release(
+    tarball_type: TarBallType,
     version: str,
```

Comment:
```suggestion
    version: str | None,
```

Reply:
This method already checks if the `version` is `None` or not. It can accept `None`
```

        if not version:
            version = get_task_sdk_version()
            console_print(f"\n[info]Using {version} retrieved from task-sdk as tarball version\n")
```
https://github.com/apache/airflow/pull/57901/files#diff-c8c21c36ecce9680097c2ac5939f73b765361894551492cf2fc65770bb25899aR316-R318